cmd/bosun/main.go

```diff
@@ -284,6 +283,9 @@ func watch(root, pattern string, f func()) {
 				if wait.After(time.Now()) {
 					continue
 				}
+				if event.Name == "web/static.go" {
```

Comment:
Stops the server from killing itself after esc runs. This was debugging that can probably be removed.